server/src/main/kotlin/suwayomi/tachidesk/global/GlobalAPI.kt

```diff
@@ -22,6 +23,10 @@ object GlobalAPI {
         path("settings") {
             get("about", SettingsController.about)
             get("check-update", SettingsController.checkUpdate)
+            path("global-update") {
+                get("", GlobalUpdateController.getSettings)
+                patch("", GlobalUpdateController.modifySetting)
+            }
```

Comment:
this is not global API, global API namespace was made to "general" app API that would not belong to either manga or anime. this used to make more sense when we had three namespaces
`suwayomi.tachidesk.global`, `suwayomi.tachidesk.manga`, `suwayomi.tachidesk.anime`

move this and friends to manga.